mmcv/ops/csrc/pytorch/cuda/scatter_points_cuda.cu

```diff
@@ -26,10 +26,15 @@ std::vector<at::Tensor> DynamicPointToVoxelForwardCUDAKernelLauncher(
   std::tie(out_coors, coors_map, reduce_count) =
       at::unique_dim(coors_clean, 0, true, true, true);
 
-  // the first element of out_coors is always (-1,-1,-1) and should be removed
-  out_coors = out_coors.slice(0, 1);
-  reduce_count = reduce_count.slice(0, 1).to(torch::kInt32);
-  coors_map = coors_map.to(torch::kInt32) - 1;
+  if (out_coors.index({0, 0}).lt(0).item<bool>()) {
```

Comment:
is there any other cases that need this judgment?

Reply:
This judgment is necessary if the input may contain out-of-range `coors` (full with -1). If we could guarantee that the input coors are all valid, then we may safely remove this branch.

Reply:
I just check the code of dynamic voxelization and found that we could not rule out the possibility that the input contains out-of-range coors.

Reply:
sure, then we can suggest @wHao-Wu to leave a comment here as a reminder.

Reply:
OK. I will leave a comment here. Thanks for your help.